frontend/public/components/_dropdown.scss

```diff
@@ -238,6 +238,8 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
   border-width: 0 !important; // TEMP fix until https://github.com/patternfly/patternfly-next/issues/2019 is fixed upstream
   list-style: none;
   -webkit-overflow-scrolling: touch;
+  max-height: 60vh;
+  overflow-y: auto;
```

Comment:
We try to keep the CSS rules alphabetized, so these would go after `list-style` above.

Reply:
ok, will update together with following comment